.travis.yml

```diff
@@ -0,0 +1,15 @@
+dist: xenial
```

Comment:
Why you need travis here? there's already a builder bot that tests every PR you make to the repository once created.

Reply:
em... I used to test this build using travis. REMOVED.